### ISSUE TITLE
Handle expired recipient key gracefully

### ISSUE THREAD

**Issue by dominikschulz** [gopasspw/gopass#117]
If one of a stores recipients public keys expires the store becomes unuseable unless this recipient is removed.

Maybe we can handle this situation more gracefully, e.g. offering to omit the invalid key on a case by case base.

**Comment by metalmatze** [comment]
We (@tclass and me) just had the problem again, that the store was unusable, due to my key being still expired (outdated) on his machine.
When inserting it didn't ask anymore if we want to encrypt for other keys, it just failed completely. (This is why I'm reopening this issue)

My key was imported while running `gopass recipients` but sadly the imported key was outdated. What happens with keys when they're expired but still in the `.gpg-keys` sub folder of each store? (This might be a separate issue)

**Comment by AnomalRoil** [comment]
I recently had this problem, and encryption failed with:
```
gpg: 9B5242E1A9D71F7C1D06E4DDF578253676181666: skipped: Unusable public key
gpg: [stdin]: encryption failed: Unusable public key
```
So I took a look at my key and saw my encryption subkey had expired.

I ended up renewing my subkeys by creating new subkeys, and had to remove my old recipient's key and add my key, I couldn't replace my old key by a new one in one complete re-encryption pass (which could be a good feature to have).

This is still actual. The problem I had was that I couldn't use:
 
- edit 
-  insert
-  generate

as long as I had an expired key in my recipients' list.
